javascript/javascript3/week3/shopping_cart/main.js

```diff
@@ -0,0 +1,104 @@
+class Product {
+  constructor(name, price) {
+    this.name = name;
+    this.price = price;
+  }
+  convertToCurrency(currency) {
+    return new Promise((resolve, reject) => {
```

Comment:
Nice, however you are not using the reject param in your promise, maybe you could implement also a scenario where the promise will be rejected and trow an error.

Reply:
@Falafelqueen oh, I should do that. Thanks.